commands/start.js

```diff
@@ -16,10 +18,20 @@ function startCmd(opts) {
 		});
 		out.success('Successfully launched snapstub server on: ' +
 			'http://localhost:' + port);
+		printRoutes(mockFolderName, port);
 	} catch (e) {
 		out.error('Failed to launch snapstub server');
 	}
 }
 
+function printRoutes(srcPath, port) {
+	out.message('\n-- Routes --');
+	fs.readdirSync(srcPath).forEach(dir => {
```

Comment:
can you please replace `fs.readdirSync` with [globby](https://github.com/sindresorhus/globby) ?